src/Core/Models/Mail/LicenseExpiredViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Bit.Core.Models.Mail
+{
+    public class LicenseExpiredViewModel : BaseMailModel
+    {
+        public bool IsOrganization { get; set; }
```

Comment:
Maybe pass in a string, OrganizationName, and include that in the email text. You can still leave this IsOrganization property for the email template conditional, but just make it a derived getting with no setter.  Something like:

```
public bool IsOrganization => !string.IsNullOrWhiteSpace(OrganizationName);
```

Reply:
This was done originally but removed as part of the simplification.  Can you provide a quick sample on how you want the org name represented in the text?

Reply:
> This email is to notify you that your Bitwarden organization license for **Organization Name** has expired and must be updated for continued use. See the following article for details about replacing your license file: